projects/orangecrab/src/Orangecrab/Domain.hs

```diff
@@ -0,0 +1,23 @@
+{-|
+Module      : Domain
+Copyright   : Copyright © 2024 QBayLogic B.V.
+License     : MIT
+Maintainer  : QBayLogic B.V.
+Stability   : experimental
+Portability : POSIX
+
+OrangeCrab / Lattice ECP5-85F specific clock domains.
+-}
+
+{-# LANGUAGE NumericUnderscores #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+module Orangecrab.Domain where
+
+import Clash.Prelude
+
+-- | 48 MHz oscillator clock of the OrangeCrab board.
+createDomain vSystem
```

Comment:
I think synchronous resets are usually used on Lattice ECP5?

Reply:
Honestly, I have no idea. So I'll look into it :)